From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Bun K Tan <bun.k.tan@intel.com>
Date: Thu, 17 Jan 2019 00:45:10 +0000
Subject: [PATCH] Fix waagent error caused in ClearLinuxDeprovisionHandler

---
 azurelinuxagent/pa/deprovision/clearlinux.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/azurelinuxagent/pa/deprovision/clearlinux.py b/azurelinuxagent/pa/deprovision/clearlinux.py
index 59690a6..3eb32a5 100644
--- a/azurelinuxagent/pa/deprovision/clearlinux.py
+++ b/azurelinuxagent/pa/deprovision/clearlinux.py
@@ -24,8 +24,8 @@ from azurelinuxagent.pa.deprovision.default import DeprovisionHandler, \
 # pylint: enable=W0611
 
 class ClearLinuxDeprovisionHandler(DeprovisionHandler):
-    def __init__(self, distro):  # pylint: disable=W0231
-        self.distro = distro
+    def __init__(self):
+        super(ClearLinuxDeprovisionHandler, self).__init__()
 
     def setup(self, deluser):
         warnings, actions = super(ClearLinuxDeprovisionHandler, self).setup(deluser)
