From 2ffb854d399c7db07671ec8d1e5f47e040818e46 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Mon, 15 Aug 2016 22:10:59 +0000
Subject: [PATCH 2/2] Update resource disk handling for new sfdisk

---
 azurelinuxagent/daemon/resourcedisk/default.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/azurelinuxagent/daemon/resourcedisk/default.py b/azurelinuxagent/daemon/resourcedisk/default.py
index d2e400a..5e3bff5 100644
--- a/azurelinuxagent/daemon/resourcedisk/default.py
+++ b/azurelinuxagent/daemon/resourcedisk/default.py
@@ -120,11 +120,11 @@ class ResourceDiskHandler(object):
         else:
             logger.info("GPT not detected")
             logger.info("Check fstype")
-            ret = shellutil.run_get_output("sfdisk -q -c {0} 1".format(device))
-            if ret[1].rstrip() == "7" and fs != "ntfs":
+            ret = shellutil.run_get_output("sfdisk -q --part-type {0} 1".format(device))
+            if ret[1].strip() == "7" and fs != "ntfs":
                 logger.info("The partition is formatted with ntfs")
                 logger.info("Format partition: {0} with fstype {1}",partition,fs)
-                shellutil.run("sfdisk -c {0} 1 83".format(device))
+                shellutil.run("sfdisk --part-type {0} 1 83".format(device))
                 shellutil.run("mkfs." + fs + " " + partition + " -F")
 
         logger.info("Mount resource disk")
-- 
2.9.2

