From aac3712908cc18b57796465388f5397520254f54 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Fri, 17 Jun 2016 17:55:33 +0000
Subject: [PATCH 2/2] Fix service for running on Clear Linux

---
 init/waagent.service | 1 -
 1 file changed, 1 deletion(-)

diff --git a/init/waagent.service b/init/waagent.service
index f465f79..0fbcb65 100755
--- a/init/waagent.service
+++ b/init/waagent.service
@@ -3,7 +3,6 @@ Description=Azure Linux Agent
 After=network.target
 After=sshd.service
 ConditionFileIsExecutable=/usr/sbin/waagent
-ConditionPathExists=/etc/waagent.conf
 
 [Service]
 Type=simple
-- 
2.9.0

