From b6f4136e67fe2a481606037e55e24661175b9d8a Mon Sep 17 00:00:00 2001
From: Robert Nesius <robert.a.nesius@intel.com>
Date: Tue, 11 Jul 2017 08:34:24 +0000
Subject: [PATCH] Update setup.py for os-release change in Clear Linux OS.

---
 setup.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 094abb5..a8bc11f 100755
--- a/setup.py
+++ b/setup.py
@@ -106,7 +106,8 @@ def get_data_files(name, version, fullname):
         set_udev_files(data_files)
         set_files(data_files, dest="/usr/share/oem",
                   src=["init/coreos/cloud-config.yml"])
-    elif name == 'clear linux software for intel architecture':
+    elif name == 'clear linux os for intel architecture' \
+            or 'clear linux software for intel architecture':
         set_bin_files(data_files, dest="/usr/bin")
         set_conf_files(data_files, dest="/usr/share/defaults/waagent",
                        src=["config/clearlinux/waagent.conf"])
-- 
2.13.2

